### PR TITLE
Fy23 redesign

### PR DESCRIPTION
**Pull request by cyndy-llnl**
Individual comments by file. In general, changes include:

-Addition of a metadescription
-Addition of framing copy (/visualization pages only)
-Reorganization of header hierarchy 
-Copy updates for fy23-redesign

### CODE REVIEW

File: pages/about/faq.md [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
 
 * Table of Contents
 {:toc}
 
-### How do I set up a GitHub account?
+<!-- START: Accordions Each h2 below will be a separate accordion. If any of the requested functionality (e.g., info boxes) is a problem, let me know and I can adjust -->
+## How Do I Set up a GitHub Account?
 
-If you’re new to GitHub and open source in general, figuring out how to get set up can be a challenge. You may want to read through the GitHub Help pages on [setting up and managing your GitHub profile](https://help.github.com/categories/setting-up-and-managing-your-github-profile/).
+If you're new to GitHub and open source in general, figuring out how to get set up can be a challenge. You may want to read through the GitHub Help pages on [setting up and managing your GitHub profile](https://help.github.com/categories/setting-up-and-managing-your-github-profile/){% raw %}{% endraw %}.
```
Comment (IanLee1521) [inline pages/about/faq.md:21]: what's the rationale for `{% raw %}{% endraw %}.` ?

File: pages/about/faq.md [hunk 2]
```diff
@@ -1,26 +1,30 @@
 ---
 title: FAQ
 layout: info
+description: Get answers to frequently asked questions about Lawrence Livermore National Laboratory's open source software.
 permalink: /about/faq/
 ---
 
-## {{ page.title }}
```
Comment (IanLee1521) [inline pages/about/faq.md:7]: I'll need to fire this up locally and check it out, I seem to recall there being a reason for starting at H2 here in these files, but I can't quite remember.

File: pages/about/faq.md [hunk 3]
```diff
@@ -29,160 +33,177 @@ If you’re new to GitHub and open source in general, figuring out how to get se
...
-* NO Export Controlled
-* NO Official Use Only (OUO)
-* NO Health Insurance Portability and Accountability Act (HIPAA)
-* NO Personally Identifiable Information (PII)
-* NO NDA or vendor-proprietary information
-* NO Unclassified Controlled Information (UCI)
-* NO Unclassified Controlled Nuclear Information (UCNI)
+* No classified information
+* No export controlled information (ECI)
+* No Health Insurance Portability and Accountability Act- (HIPAA-) related information
```
Comment (IanLee1521) [inline pages/about/faq.md:97]:
```suggestion
* No Health Insurance Portability and Accountability Act (HIPAA) related information
```

File: pages/about/faq.md (hunk 1 above)
Comment (IanLee1521) [reply pages/about/faq.md:21]: Note that I see this in many places (every link?), and don't understand why it's there.

File: pages/about/index.md [hunk 4]
```diff
@@ -1,62 +1,76 @@
...
+## Other Resources
+<!-- START: Quicklinks boxes -->
+* [18F Handbook: GitHub](https://handbook.18f.gov/github/){% raw %}{% endraw %}
+* [Controls necessary for Federal Use of GitHub](https://github.com/fisma-ready/github){% raw %}{% endraw %}
+* [Guidance for Agency Use of Third-Party Websites and Applications](https://obamawhitehouse.archives.gov/sites/default/files/omb/assets/memoranda_2010/m10-23.pdf){% raw %}{% endraw %}
+* [Facts about Publishing Open Source Code](https://18f.gsa.gov/2016/08/08/facts-about-publishing-open-source-code-in-government/){% raw %}{% endraw %}
+<!-- END: Quicklinks boxes -->
+
+## Contact
+For more information, check out our [FAQ](faq). If you have other questions or would like to request a private repository, please don't hesitate to contact [Ian Lee](mailto:ian@llnl.gov) or our [LLNL GitHub admins](mailto:github-admin@llnl.gov). You can also find us on our mailing list: [open-source@llnl.gov](mailto:open-source@llnl.gov).
```
Comment (IanLee1521) [inline pages/about/index.md:73]: While we are here:

```suggestion
For more information, check out our [FAQ](faq). If you have other questions or would like to request a private repository, please don't hesitate to contact our [LLNL GitHub admins](mailto:github-admin@llnl.gov). You can also find us on our mailing list: [open-source@llnl.gov](mailto:open-source@llnl.gov).
```

**Review by IanLee1521 — CHANGES_REQUESTED** [review]
@cyndy-llnl - this looks great overall! Really thankful for your work here!

The biggest thing I want to hash out is the presence of the `{% raw %}{% endraw %}` blocks that you've added, why are those being added after all the links?

**Comment by cyndy-llnl** [comment]
> @cyndy-llnl - this looks great overall! Really thankful for your work here!
> 
> The biggest thing I want to hash out is the presence of the `{% raw %}{% endraw %}` blocks that you've added, why are those being added after all the links?

Hi Ian! That's based on guidance from [digital.gov](https://designsystem.digital.gov/components/link/) around the handling of external links. The {% raw %}{% endraw %} adds an external link icon on dev.llnl.gov, but perhaps that isn't true for the Software site? 

I should also note: Some of these updates "won't make sense" until our dev has had the chance to get in and implement the design (coming soon!) stuff. We know what components we're building so I went ahead and prepped the copy accordingly, but the full thing won't be complete for a hot minute.

**Comment by hauten** [comment]
I will set aside some time very soon to review in detail locally

File: pages/about/index.md [hunk 5]
```diff
@@ -1,62 +1,76 @@
...
+
+### *Science & Technology Review* coverage
+<!-- START: Quicklinks boxes -->
+* [Ambassadors of Code (cover story)](https://str.llnl.gov/2018-01/lee)
+* [Ambassadors of Code (video, 3:04)](https://youtu.be/nTxMn1NWHQU){% raw %}{% endraw %}
+* [Commentary by Bruce Hendrickson](https://str.llnl.gov/2018-01/comjan18), LLNL Computing associate director
+* [Visualization Software Stands the Test of Time](https://str.llnl.gov/2021-05/brugger)
+<!-- END: Quicklinks boxes -->
+  
+### 2020 LLNL Computing Virtual Expo Materials
```
Comment (hauten) [inline pages/about/index.md:57]: @cyndy-llnl What do you think about removing "2020" from this heading so it doesn't seem so outdated? The assets from the Expo are fairly evergreen.

**Review by hauten — COMMENTED** [review]
@cyndy-llnl If by chance the changes go live before the end of August, I'd like to retain the temporary banner from the currently live home page. It advertises tutorials that end on Aug 31. After that date, we don't need it anymore. Markup for the banner is in https://github.com/LLNL/llnl.github.io/pull/615/commits/47f696af130cdd01fc301fc739dea8a046799af2.

**Comment by hauten** [comment]
@cyndy-llnl and @IanLee1521 I'm finished reviewing. I have one question for Cyndy above, plus one note about the temporary tutorials advertisement on the currently live home page. I made two commits that Cyndy may want to review. No other changes from me.

File: pages/about/index.md (hunk 5 above)
Comment (cyndy-llnl) [reply pages/about/index.md:60]: @hauten That works! I'm a fan of including the date if something is "of a moment," but when it's evergreen, you'll entice more folks to use it when an older date is omitted. 

One thing: Is that a change you're making or that you'd like me to make? Happy to participate however you prefer!

**Comment by cyndy-llnl** [comment]
@hauten We made a note about the tutorial banner and will retain it if we're able to launch before 8/31. I think the commits look good (and they'll make even better sense once Doug gets ahold of the site). Thanks!

File: pages/about/index.md (hunk 5 above)
Comment (hauten) [reply pages/about/index.md]: @cyndy-llnl I made the change just now, thx

**Comment by hauten** [comment]
Incorporated and confirmed as part of https://github.com/LLNL/llnl.github.io/pull/647

**Pull request closed without merging**